packages/order-experience-api/src/api/createController.ts

```diff
@@ -82,6 +83,14 @@ export class CreateController extends AbstractController<typeof requestSchema> {
       `Create Order with Items for namespace ${namespace} and user ${user}`
     )
 
+    // Set merchant id for items from corresponding product mapping
+    for (const item of items) {
+      const productMapping = await getProductMapping({
+        productId: item.productId,
+      })
+      if (productMapping) item.merchantId = productMapping.merchantId
```

Comment:
Do not use one line if statements, change to ternary or add code inside brackets

Reply:
Good point, the ternary would be best in this case.